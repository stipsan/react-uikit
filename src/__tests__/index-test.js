it('should export', () => {
  expect(Object.keys(require('../index'))).toMatchSnapshot()
})
