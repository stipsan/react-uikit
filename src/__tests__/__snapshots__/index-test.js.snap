// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should export 1`] = `
Array [
  "Button",
  "Dropdown",
  "Modal",
  "Notification",
  "Input",
]
`;
